### PR TITLE
refactor: with empty token

### PR DESCRIPTION
**Pull request by zombieJ**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [x] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |      Refactor Empty with full token.     |
| 🇨🇳 Chinese |       -    |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/empty/style/index.tsx [hunk 1]
```diff
@@ -102,30 +97,24 @@ const genSharedEmptyStyle: GenerateStyle<EmptyToken> = (token): CSSObject => {
 
       // 原来 &-footer 没有父子结构，现在为了外层承担我们的hashId，改成父子结果
       [`${componentCls}-footer`]: {
-        // FIX ME
-        marginTop: '16px',
+        marginTop: marginXS * 2,
```
Comment (MadCcc) [inline components/empty/style/index.tsx:100]: 这个为啥不是 margin？

File: components/empty/style/index.tsx [hunk 2]
```diff
@@ -18,7 +19,7 @@ const genEmptyImgStyle = (token: EmptyToken): CSSObject => {
   const emptyImgSimpleCls = `${emptyImgCls}-simple`;
 
   return {
-    // FIX ME have not theme
+    // FIXME: have not theme
```
Comment (MadCcc) [inline components/empty/style/index.tsx:22]: theme 相关的 可以去掉了吧

Comment (zombieJ) [reply components/empty/style/index.tsx:22]: 这个先留着，还需要和 闻冰 讨论。Image 的必要特殊，没办法用 token 算出来。

File: components/empty/style/index.tsx [hunk 3]
```diff
@@ -102,30 +97,24 @@ const genSharedEmptyStyle: GenerateStyle<EmptyToken> = (token): CSSObject => {
...
-        // FIX ME
-        marginTop: '16px',
+        marginTop: margin,
       },
 
       '&-normal': {
-        // FIX ME
-        margin: '32px 0',
-        // '@disabled-color'
+        marginBlock: margin * 2,
```
Comment (MadCcc) [inline components/empty/style/index.tsx:104]: marginXL 是不是应该是 32，marginXXL 是 48。。。

**Review by MadCcc — APPROVED** [review]

**Pull request merged**